packages/data-explorer/src/views/EntityView.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <div class="mt-1 entity-table" v-if="activeEntityData && activeEntityData.items.length > 0 && entityMeta">
+    <div v-if="isShop && entitiesToShow.length === 0" class="alert alert-warning">
+      {{ 'dataexplorer_empty_shopping_cart' | i18n}}
+    </div>
+
+    <div class="card-columns" v-else-if="dataDisplayLayout==='cards'">
+      <entity-card
```

Comment:
maby move to separate view components and use the dynamic component stuff instead of if else, then again this seems fine for now

Reply:
We merged this on purpose because the entitycards and entityrows are different views that use exactly the same methods and functions. We merged this to prevent code duplication.